specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -4003,6 +4003,215 @@
           }
         }
       }
+    },
+    "/{scope}/providers/Microsoft.Resources/tags/default": {
+      "put": {
+        "tags": [
+          "Tags"
+        ],
+        "operationId": "Tags_ResourceCreate",
+        "description": "Create or Replace existing tags with pass-in tags.",
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope of the resources.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
```

Comment:
Reference the shared "ApiVersionParameter" definition

Reply:
Referenced ApiVersionParameter in parameters